ambry-cloud/src/main/java/com.github.ambry.cloud/CloudBlobStore.java

```diff
@@ -466,6 +466,11 @@ public boolean isEmpty() {
     return false;
   }
 
+  @Override
+  public boolean isBootstrapInProgress() {
+    throw new UnsupportedOperationException("Method not supported");
```

Comment:
This method within CloudBlobStore is not supposed to be called in practice but in case logic is changed I will return false directly because there is no bootstrap stage for `CloudBlobStore`.